Documentation/BuildInstructions.md

```diff
@@ -185,12 +185,15 @@ $ cd Toolchain
 $ ./BuildIt.sh
 ```
 
-Building the toolchain will also automatically create a `Build/i686/` directory for the build to live in.
+Building the toolchain will also automatically create a `Build/i686/` directory for the build to live in. Specifically, change the current directory to the `Build/` parent directory. To go there from `Toolchain/`, use this command:
+
+```console
+$ cd ../Build/i686
+```
 
 Once the toolchain has been built, go into the `Build/i686/` directory and run the commands. Note that while `ninja` seems to be faster, you can also just use GNU make, by omitting `-G Ninja` and calling `make` instead of `ninja`:
```

Comment:
> go into the `Build/i686/` directory

This shouldn't be mentioned *after* cd'ing :)

Reply:
I missed that part. (:

Reply:
Made second commit to reflect changes.